Silicon/TigerlakePkg/Library/VTdLib/VTdLib.c

```diff
@@ -117,7 +117,7 @@ UpdateDrhd (
     NeedRemove = TRUE;
   }
   if ((DrhdEngine->DeviceScope[0].PciPath.Device == SA_IGD_DEV) && (DrhdEngine->DeviceScope[0].PciPath.Function == SA_IGD_FUN_0) &&
-      (PciRead32 (PCI_LIB_ADDRESS (0, DrhdEngine->DeviceScope[0].PciPath.Device, DrhdEngine->DeviceScope[0].PciPath.Function, 0x00)) == 0xFFFFFFFF)) {
+      (PciRead32 (PCI_LIB_ADDRESS (0, DrhdEngine->DeviceScope[0].PciPath.Device, DrhdEngine->DeviceScope[0].PciPath.Function, 0x00)) != 0xFFFFFFFF)) {

```

Comment:
This line is too long.  Consider using a variable VidDid instead.

Reply:
sure. will update